src/start/PasswordAuthView.js

```diff
@@ -95,14 +109,21 @@ class PasswordAuthView extends React.Component {
           onChangeText={newEmail => this.setState({ email: newEmail })}
         />
         <Input
-          style={styles.field}
+          style={styles.passwordfield}
           placeholder="Password"
-          secureTextEntry
+          secureTextEntry={this.state.ishidden}
           value={password}
           onChangeText={newPassword => this.setState({ password: newPassword })}
           blurOnSubmit={false}
           onSubmitEditing={this.validateForm}
         />
+        <View style={styles.togglePassword}>
```

Comment:
A better approach would be to have a `PasswordInput` component that uses `Input` and it contains the Show/Hide toggle inside.